docs/source/guides/development/cli_guide.rst

```diff
@@ -5,14 +5,16 @@ CLI Guide
 Overview
 --------
 
-.. important::
-
-    The guide uses NuCypher's Görli Testnet.
-
 This guide is an example of NuCypher's decentralized network allowing Alice to share a secret
 with Bob using the NuCypher Network via the ``nucypher`` CLI. It is analogous to the ``python`` example
 in :doc:`getting_started`.
 
+.. note::
+
+    While the example provided uses Ethereum mainnet, these steps can be followed for the Goerli Testnet
+    with updated `geth` (``~/.ethereum/goerli/geth.ipc``) and `seed` uris
```

Comment:
Should we note the Goerli seed uri here?

Reply:
✔️